src/lib.rs

```diff
@@ -59,12 +59,14 @@ impl Bencher {
     ///
     /// # Timing loop
     ///
-    /// ``` ignore
-    /// start = Instant::new();
+    /// ```rust,no_run
+    /// # fn routine() {}
+    /// let iters = 4_000_000;
```

Comment:
Hide this variable with `#`. The number of iterations is chosen by the `Bencher` at runtime, this gives the impression that `iters` is fixed.